idl/chromadb/proto/chroma.proto

```diff
@@ -114,7 +114,7 @@ message QueryMetadataRequest {
     string segment_id = 1;
     Where where = 2;
     WhereDocument where_document = 3;
-    repeated string ids = 4;
+    optional UserIds ids = 4;
```

Comment:
I don't think there is but is there anywhere else we expect option<vec<string>>? from a proto? we should make the same change elsewhere then.

Reply:
I've noticed that `GetVectorsRequest` and `QueryVectorsRequest` accept a sequence of string ids, similar to `QueryMetadataRequest`. In the frontend, `get_vectors` and `query_vectors` in `grpc_sement.py` accepts `Optional[Sequence[str]]` for these fields, but the caller of these methods explicitly check for the case of empty ids, and the query node is expecting `Vec<String>` as inputs. I'm unsure whether we should convert these into optional fields as well.

Reply:
```
message GetVectorsRequest {
    repeated string ids = 1;
    string segment_id = 2;
    string collection_id = 3;
}
```

I think ids is not-optional here but have to check.